backend/src/main/kotlin/no/bekk/routes/UserInfoRouting.kt

```diff
@@ -1,13 +1,43 @@
 package no.bekk.routes
 
+import io.ktor.http.*
 import io.ktor.server.application.*
 import io.ktor.server.response.*
 import io.ktor.server.routing.*
 import no.bekk.authentication.getGroupsOrEmptyList
+import no.bekk.authentication.getCurrentUser
+import no.bekk.services.MicrosoftGraphService
+import no.bekk.util.logger
 
 fun Route.userInfoRouting() {
-    get("/userinfo"){
-        val groups = getGroupsOrEmptyList(call)
-        call.respond(mapOf("groups" to groups))
+    val microsoftGraphService = MicrosoftGraphService()
+    route("/userinfo") {
+        get {
+            val groups = getGroupsOrEmptyList(call)
+            call.respond(mapOf("groups" to groups))
+        }
+
+        get("/currenUser") {
+            val user = getCurrentUser(call)
+            call.respond(HttpStatusCode.OK, user)
+        }
```

Comment:
```suggestion
        get {
            val groups = getGroupsOrEmptyList(call)
            val user = getCurrentUser(call)
            call.respond(mapOf("groups" to groups, "user" to user))
        }
```

Kan disse slåes sammen?

Reply:
Fikset!